Resources/Prototypes/Entities/Objects/Tools/gas_tanks.yml

```diff
@@ -69,19 +69,6 @@
   - type: Clothing
     sprite: Objects/Tanks/oxygen.rsi
 
-- type: entity
```

Comment:
As discussed, please don't remove this; OR PR its removal in a separate change so that forks can easily revert it if they need it.